src/components/SelectBox/SelectBox.tsx

```diff
@@ -194,7 +199,7 @@ const getLabelForValue = (
 const getImgForValue = (
   options: KaSelectBoxOptionListType[],
   value: string,
-): string | undefined => {
+): React.ReactNode | undefined => {
```

Comment:
Since the return type can be either `option.img` or `undefined`, it seems fine to change it to `string | ReactElement | undefined`.